src/Microsoft.Graph.Core/Requests/Content/BatchRequestContent.cs

```diff
@@ -78,8 +78,46 @@ public BatchRequestContent(params BatchRequestStep[] batchRequestSteps)
         /// <returns>True or false based on addition or not addition of the provided <see cref="BatchRequestStep"/>. </returns>
         public bool AddBatchRequestStep(BatchRequestStep batchRequestStep)
```

Comment:
When we do a breaking change we should consider changing this to return void and throw an exception rather than return bool.  Requiring a consumer to check a return value is a `C` style thing to do and not something a C# dev would expect.

Reply:
New issue created to track this https://github.com/microsoftgraph/msgraph-sdk-dotnet-core/issues/49